apis/nucleus/src/object/lo-handler.js

```diff
@@ -60,7 +62,7 @@ export default function loHandler({ dc: lo, def, properties }) {
       return 0;
     },
     canAddDimension() {
-      return handler.dimensions().length === 0;
+      return handler.dimensions().length < handler.maxDimensions();
```

Comment:
Still should still be strickly for zero dims right? Else you'd end up in the other handler, so a change here is just more confusing? or?

Reply:
Yes, I will fix.

Reply:
For the Listbox case we can always assume 1 dim. But we still align to the API where maxDims has to return a value. Therefore, one could argue either way here too but let's be explicit and set it to `0` 😄